src/table/components/virtualized-table/TableContainer.tsx

```diff
@@ -85,4 +85,9 @@ export default function TableContainer(props: TableContainerProps) {
       </FullSizeContainer>
     </div>
   );
-}
+};
+
+// Export non memoized version for testing purpose
+export { TableContainer };
```

Comment:
for the record it is possible to test when you export memo(someComponent) but the syntax is weird so I like this better actually